include/circt/Support/LLVM.h

```diff
@@ -131,6 +131,7 @@ class TypeConverter;
 class TypeID;
 class TypeRange;
 class TypeStorage;
+class UnitAttr;
```

Comment:
Feel free to push the LLVM.h changes separately, they're obvious and unrelated to the rest of this patch.